.remarkrc.js

```diff
@@ -0,0 +1,12 @@
+// File for setting up the markdown linter configuration.
```

Comment:
The descriptions should be ready for junior developers who don’t remember `linter` and `Markdown` words.

Something like:

```
The config of tool to check documentation in .md files for popular mistakes
```